ClientReceiveItems.py

```diff
@@ -81,7 +81,7 @@ async def handle_disable_gravity_suit(ctx: 'MetroidPrimeContext', current_items:
 
 async def handle_receive_missiles(ctx: 'MetroidPrimeContext', current_items: dict[str, InventoryItemData]):
     # Slot data is required for missiles + Power Bombs so we can check if launcher / main pb are enabled
-    if ctx.slot_data is not None:

```

Comment:
Can we separate these optimizations into a separate PR? I want to be able to test them separately

Reply:
Ah I thought they were more expansive, I can test these. If you could use the SuitUpgrade enum value instead of strings for these that'd be 🔥

Reply:
Ah disregard that as well, I can just add that post merge. I saw that these were already not using the enum